Avatar table

barankyle opened the pull request:
## Summary

Making a discrete avatar table to centralize avatar information, instead of having to store extra
information on the static-resource table and do awkward queries on it.


## References

closes #_insert number here_


## Checklist
- [x] If this PR is still a WIP, convert to a draft
- [x] [ensure all checks pass](https://github.com/XRFoundation/XREngine/wiki/Testing-&-Contributing)
- [x] When this PR is ready, mark it as "Ready for review"
- [x] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer


## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

speigg reviewed (commented):
Looks like upload-asset service has a dependency on the avatar service; shouldn't this be reversed?

barankyle commented:
> Looks like upload-asset service has a dependency on the avatar service; shouldn't this be reversed?

I think I corrected this in later changes (I've squashed everything down to one commit).
